src/utils/web3-provider/methods/eth_sendTransaction.js

```diff
@@ -28,11 +28,13 @@ export default async ({ payload, store, requestManager }, res, next) => {
     };
   }
   let currency = store.getters['external/contractToToken'](tx.to);
+  const gasPriceByTypeGetter = store.getters['global/gasPriceByType'];
+  const gasPriceType = store.state['global/gasPriceType'];
   if (!currency)
     currency = store.getters['external/contractToToken'](MAIN_TOKEN_ADDRESS);
   tx.gasPrice = tx.gasPrice
     ? tx.gasPrice
-    : BigNumber(store.getters['global/gasPrice']).toFixed();
```

Comment:
`global/gasPrice` is the user selected gasPrice this whole thing is redundant. Look at the gasprice getter implementation
https://github.com/MyEtherWallet/MyEtherWallet/blob/main/src/core/store/global/getters.js#L47